### PR TITLE
Removes duplicate private declaration

### PR DESCRIPTION
**Pull request by brunoarueira**

### CODE REVIEW

**Review by javierjulio — APPROVED** [review]
Thanks! ❤️

**Review by deivid-rodriguez — COMMENTED** [review]
Thanks @brunoarueira!

Can you enable the `Lint/UselessAccessModifier` rubocop cop? Just to double check more instances of this issue and make sure it doesn't happen again in the future. It'd be a matter of explicitly enabling the cop in `.rubocop.yml` and then run `rubocop` and autocorrect whatever it finds.

**Comment by brunoarueira** [comment]
@deivid-rodriguez Sure, I'll do and update asap!

**Comment by deivid-rodriguez** [comment]
Thanks!

**Comment by brunoarueira** [comment]
@deivid-rodriguez done!

File: lib/active_admin/error.rb [hunk 1]
```diff
@@ -45,8 +45,6 @@ def self.capture
       raise new exception
     end
 
-    private
```
Comment (deivid-rodriguez) [inline lib/active_admin/error.rb:48]: Interesting. I think the intention here was to make the `database_error_classes` method private. Should we add `private_class_method :database_error_classes` here to make that happen?

File: lib/active_admin/resource/routes.rb [hunk 2]
```diff
@@ -42,8 +42,6 @@ def route_uncountable?
         config[:route_collection_name] == config[:route_instance_name]
       end
 
-      private
```
Comment (deivid-rodriguez) [inline lib/active_admin/resource/routes.rb:45]: In this case, it seems like the intention was to make the `RouteBuilder` class private, but it doesn't work like that. Should we add `private_constant :RouteBuilder` to make that actually happen?

File: lib/active_admin/resource_controller/decorators.rb [hunk 3]
```diff
@@ -57,8 +57,6 @@ def self.wrap(decorator)
           @cache[name] ||= wrap! collection_decorator, name
         end
 
-        private
```
Comment (deivid-rodriguez) [inline lib/active_admin/resource_controller/decorators.rb:60]: Same here, it seems like the code wanted to make `wrap!` private, but it doesn't work like that.

**Review by deivid-rodriguez — COMMENTED** [review]
It turns out that the duplicate `private` you removed was the only case of a duplication. The other cases seem to be trying to make methods/constants private in an incorrect way.

I guess we could leave this PR as it is, but open a new ticket to actually make these things private? Or maybe just change it here, I just don't want to break anybody's apps, but it seems very unlikely that anybody is actually using these.

@javierjulio What do you think?

**Comment by deivid-rodriguez** [comment]
@brunoarueira Sorry, not sure how to proceed here, a simple PR got complicated :laughing:

**Review by deivid-rodriguez — APPROVED** [review]
I'd say let's stay on the safe side and just merge this PR as it is now?

**Comment by brunoarueira** [comment]
> @brunoarueira Sorry, not sure how to proceed here, a simple PR got complicated

@deivid-rodriguez Yeah, when I see those changes proposed by rubocop, I thought it isn't right. Bu anyway, I can make the requested changes at another pull request if you prefer.

**Comment by javierjulio** [comment]
No problem at all! Yes I think this is good. The points at which `private` was removed, it wouldn't have applied anyway since the code there has the same public access level. I think it's good to merge as is. Thanks!

**Review by javierjulio — APPROVED** [review]

**Comment by deivid-rodriguez** [comment]
Thanks @brunoarueira!!

**Pull request merged**